.travis.yml

```diff
@@ -39,6 +39,9 @@ jobs:
   - name: Functional test of the CentOS 6 template
     script: bash -x travis_ci/test.sh centos6 server
 
+  - name: Test to check if there are duplicate templates create
```

Comment:
I would reword this to "Ensure that a single template is exposed for each template variant (os/workload/size trio)"

Reply:
Done